src/signalo/value_lists/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class OfficialSignType(models.Model):
+    id = models.CharField(primary_key=True, max_length=10)
+    active = models.BooleanField(default=True)
+    value_de = models.CharField(max_length=255, null=True, blank=True)
+    value_fr = models.CharField(max_length=255, null=True, blank=True)
+    value_it = models.CharField(max_length=255, null=True, blank=True)
+    value_ro = models.CharField(max_length=255, null=True, blank=True)
+    description_de = models.TextField(null=True, blank=True)
+    description_fr = models.TextField(null=True, blank=True)
+    description_it = models.TextField(null=True, blank=True)
+    description_ro = models.TextField(null=True, blank=True)
+    img_de = models.FileField(upload_to="official_signs/de")
+    img_fr = models.FileField(upload_to="official_signs/fr")
+    img_it = models.FileField(upload_to="official_signs/it")
+    img_ro = models.FileField(upload_to="official_signs/ro")
+    img_height = models.IntegerField(default=0)
```

Comment:
this needs to be calculated. 
or for now, we can take the values from the json (maybe it's already the case)

Reply:
Yes currently there's a value specified in `officials_signs.json`. Happy to compute it though, even if I am not sure what would count as appropriate values.

Reply:
let's keep it like this for now, this is business specific and it shouldn't affect us for now.